### PR TITLE
Release 2.7.0

### PR DESCRIPTION
**Pull request by rom1504**
# Description

## Type of change

- [ ] Feature
- [ ] Fix
- [x] Release

## Checklist

- [x] I have followed the [CONTRIBUTING](https://github.com/criteo/deepr/blob/master/docs/CONTRIBUTING.rst) guidelines.
- [x] I have updated the [CHANGELOG](https://github.com/criteo/deepr/blob/master/docs/CHANGELOG.rst).

### CODE REVIEW

File: docs/CHANGELOG.rst [hunk 1]
```diff
@@ -23,6 +23,25 @@ Fixed
...
+[2.7.0] - 2020-08-04
+--------------------
+
+Added
+~~~~~
+- Add encoding support to Path / HDFSFile
+- Movielens example
+
+Changed
+~~~~~~~
```
Comment (guillaumegenthial) [inline docs/CHANGELOG.rst:35]: - Changed default initializer for embeddings in the embeddings layer

File: docs/CHANGELOG.rst [hunk 2]
```diff
@@ -23,6 +23,25 @@ Fixed
 Security
 ~~~~~~~~
 
+[2.7.0] - 2020-08-04
+--------------------
+
+Added
+~~~~~
+- Add encoding support to Path / HDFSFile
+- Movielens example
```
Comment (guillaumegenthial) [inline docs/CHANGELOG.rst:32]: - TripletPrecision layer

Comment (rom1504) [reply docs/CHANGELOG.rst:32]: done

File: docs/CHANGELOG.rst (hunk 1 above)
Comment (rom1504) [reply docs/CHANGELOG.rst:36]: done

**Pull request merged**